src/main/java/service/operations/report/ReportWriter.java

```diff
@@ -0,0 +1,14 @@
+package service.operations.report;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ReportWriter {
+    public static void writeReportToFile(String reportContent, String filePath) {
```

Comment:
No need to be specific what it writes, because it can write anything really, so let's keep more abstract and universal name. Also why static?
```suggestion
public class FileWriter {
    public void writeDataToFile(String data, String filePath) {
```